scss/grid/_grid.scss

```diff
@@ -33,6 +33,9 @@ $block-grid-max: 8 !default;
 // Internal value to store the end column float direction
 $-zf-end-float: if($grid-column-align-edge, $global-right, $global-left);
 
+// Selector used for an alias of column (with @extend). If `false`, no alias is created.
```

Comment:
The sass-doc comment must be properly formatted. Please read: sassdoc.com/annotations/

```sass
/// (...)
/// @type String
```